contracts/IdentityManager.sol

```diff
@@ -0,0 +1,219 @@
+pragma solidity 0.4.11;
+import "./Proxy.sol";
+
+
+contract IdentityManager {
```

Comment:
I can rework this over the weekend. For a name, I think MetaIdentityManager works well.

Also, you mean just define a "ManagerInterface," right? If we do inheritance, I'm not sure we can just add modifiers on to functions (aka, would have to redefine functions anyway). 

I do think it would make sense to make sure they share the same interface, though. Let me know what you think @pelle